File: src/main/java/eterea/migration/api/rest/model/ProductTransaction.java
package eterea.migration.api.rest.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@EqualsAndHashCode(callSuper = true)
@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table
public class ProductTransaction extends Auditable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long productTransactionId;
    private Long orderNumberId;
    private String nombreProducto;
    private String montoProducto;
}

File: src/main/kotlin/eterea/migration/api/rest/kotlin/model/ProductTransaction.kt
package eterea.migration.api.rest.kotlin.model

import jakarta.persistence.*

@Entity
@Table
data class ProductTransaction(

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var productTransactionId: Long? = null,
    var orderNumberId: Long? = null,
    var nombreProducto: String = "",
    var montoProducto: String? = null,

) : Auditable()
